sentry/src/main/java/io/sentry/PropagationContext.java

```diff
@@ -51,6 +51,19 @@ public static PropagationContext fromHeaders(
         sentryTraceHeader.isSampled());
   }
 
+  public static @NotNull PropagationContext fromExistingTrace(
+      final @NotNull String traceId,
+      final @NotNull String spanId,
+      final @Nullable Double decisionSampleRate,
+      final @Nullable Double decisionSampleRand) {
```

Comment:
Can we add an optional `Baggage` param here as well?

Reply:
I don't see the need or benefit of doing that from the Unity SDK's perspective. @krystofwoldrich would this help RN?

Reply:
I believe we pass replayId via baggage to downstream services
Might need that on React Native (which has Replay)

Reply:
We don't need that because the ReplayID is generated on native and synced up to RN Replay Integration which adds it to DSC. 

https://github.com/getsentry/sentry-react-native/blob/5dff5ee95fb4c3d8f2589b80d4d355fecde17778/packages/core/src/js/replay/mobilereplay.ts#L107

The traceID is the opposite, hybrid (RN) generates it and we need to sync it down to native.